static/src/javascripts/projects/common/modules/article/space-filler.js

```diff
@@ -1,21 +1,18 @@
-/* jscs:disable disallowDanglingUnderscores */
 define([
-    'common/utils/$',
-    'common/utils/fastdom-idle',
+    'common/utils/fastdom-promise',
     'Promise',
     'raven',
     'common/modules/article/spacefinder'
 ], function (
-    $,
-    idleFastdom,
+    fastdom,
     Promise,
     raven,
     spacefinder
 ) {
-    return new SpaceFiller();
+    var lastIteration;
```

Comment:
Won't this still be stateful? The reason I originally made this a private variable of the SpaceFiller module was that this way, a test could re-initialize the whole thing and be sure that it was working with a tabula rasa. I guess it won't impact the test as it stands, though.